pkg/agent/runner.go

```diff
@@ -1521,6 +1534,8 @@ func (r *Runner) doVMUpdate(
 		panic(fmt.Errorf("Unexpected VM patch request failure: %w", err))
 	}
 
+	r.global.metrics.neonvmRequestsOutbound.WithLabelValues("ok").Inc()
```

Comment:
constants for labels?

Reply:
Given this label value is only set once, in that location, I'm going to leave it